src/Files.App/Helpers/PathNormalization.cs

```diff
@@ -54,7 +54,7 @@ public static string NormalizePath(string path)
 			}
 			catch (Exception ex) when (ex is UriFormatException || ex is ArgumentException)
 			{
-				App.Logger.LogWarning(ex, path);
+				App.Logger.LogDebug(ex, path);
```

Comment:
How do you use LogInformation or LogDebug depending on the situation?

Reply:
LogInformation for non errors, and LogDebug for errors that don't need to be uploaded to Sentry, but are still nice to have locally.